crates/store/re_types/definitions/rerun/blueprint/datatypes/filter_by_event.fbs

```diff
@@ -4,8 +4,9 @@ namespace rerun.blueprint.datatypes;
 /// Configuration for the filter by event feature of the dataframe view.
 // Has to be a table because flatbuffer doesn't support strings in structs.
 table FilterByEvent (
-  "attr.rust.derive": "Default, PartialEq, Eq",
-  "attr.rerun.scope": "blueprint"
+    "attr.python.aliases": "str, blueprint_datatypes.ComponentColumnSelector",
```

Comment:
Below you used `blueprint_datatypes.ComponentColumnSelectorLike` (note the extra `Like`). I expect this should be the same here?